Updated main documentation

benventive opened the pull request:
GitHub Issue: #
<!-- Link to relevant GitHub issue if applicable.
     All PRs should be associated with an issue -->

## Proposed Changes
<!-- Please un-comment one ore more that apply to this PR -->

<!-- - Bug fix -->
<!-- - Feature -->
<!-- - Code style update (formatting) -->
<!-- - Refactoring (no functional changes, no api changes) -->
<!-- - Build or CI related changes -->
- Documentation content changes
<!-- - Other, please describe: -->


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying,
     or link to a relevant issue. -->


## What is the new behavior?
<!-- Please describe the new behavior after your modifications. -->


## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

<!-- If this PR contains a breaking change, please describe the impact
     and migration path for existing applications below. -->


## Other information
<!-- Please provide any additional information if necessary -->

Commit:
Updated main documentation

Commit:
Updated nuget metadata

MatFillion commented on README.md:
```diff
@@ -9,11 +9,17 @@ A set of MSBuild extensions to ease cross-platform development.
 
 ## Features
 
+- Run lightweight builds to make pull requests builds faster.
```
Je crois que je clarifierais quel package offre quelle feature.

benventive replied:
J'ai ajouté le nom du package en avant de chaque ligne

Commit:
Prevented symbol creation

The projects do not generate binaries, making the symbols generation parameters unnecessary.

Commit:
Added feature details
